feature-group-definitions/aborting.yml

```
@@ -1,4 +1,5 @@
 name: AbortController and AbortSignal
+description: The `AbortController` and `AbortSignal` interfaces allow aborting an ongoing operation, such as a `fetch()` request.
```

style nit: avoiding "allow" (though I'll grant that "handle" is not _great_)

```suggestion
description: The `AbortController` and `AbortSignal` interfaces handle cancelling an ongoing operation, such as a `fetch()` request.
```

---

I think "handle" sounds like these interfaces do something automatically, as oppose to enabling developers to do something. To write this as "The X aborts Y" is hard, since it's the `abort()` method on an `AbortController` instance `controller`, when `controller.signal` (an `AbortSignal` instance) has been given to the API that supports aborting. Too much into the weeds IMHO.

We'll need some pattern for cases like this common case where a complex API is passive, but allows a developer to act. Ideas:

- allows
- enables
- supports
- ???

---

That's a first for "you" but it's not uncommon to see in docs for developers. Sure, why not. I'll also go with cancel.